bids/external/inflect.py

```diff
@@ -1686,55 +1686,36 @@ def get_si_pron(thecase, word, gender):
     # 1st PERS. SING.  2ND PERS. SING.   3RD PERS. SINGULAR
     # 3RD PERS. (INDET.)
     "act": "act",
-    "act": "act",
```

Comment:
Please do not modify external modules. If you would like to see this changed, get it accepted upstream, and then open a PR to update the external module from upstream.

I think it's unlikely that you will get this change merged upstream, as it is intentionally groups of first/second/third person conjugations.

Reply:
Fixed.